charts/xui-mo-webapp/values.aat.template.yaml

```diff
@@ -16,8 +16,8 @@ nodejs:
     #S2S_SERVICE: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
     #FEE_AND_PAY_API: https://payment-api-aat.service.core-compute-aat.internal
-    MANAGE_CASE_LINK: https://xui-webapp-aat.service.core-compute-aat.internal/cases
-    MANAGE_ORG_LINK: https://xui-mo-webapp-aat.service.core-compute-aat.internal/
-    
+    MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/cases
```

Comment:
Is this desired? Replacing `aat` values with `production` values that has `aat` in its file name?